src/AngleSharp.Css.Tests/Extensions/AnalysisWindow.cs

```diff
@@ -248,6 +248,41 @@ public void GetComputedStyleMixedTrivialAndPseudoScenario()
             Assert.AreEqual(3, stylePseudo.Length);
         }
 
+        [Test]
+        public void FirstLetterPseudoElementDoesNotAffectElementComputedStyle()
+        {
+            var sourceCode = "<!doctype html><head><style>.dropcap { color: blue; } .dropcap::first-letter { color: red; }</style></head><body><p class='dropcap'>Alpha</p>";
+
+            var document = ParseDocument(sourceCode);
+            Assert.IsNotNull(document);
+            var window = document.DefaultView;
+
+            var element = document.QuerySelector(".dropcap");
+            Assert.IsNotNull(element);
+
+            var style = window.GetComputedStyle(element);
+            Assert.IsNotNull(style);
+            Assert.AreEqual("rgba(0, 0, 255, 1)", style.GetColor());
```

Comment:
Use Assert.That. 
Actual values go on the left, expectations on the right
```suggestion
            Assert.That(style.GetColor(), Is.EqualTo("rgba(0, 0, 255, 1)"));
```